Source/utils/display.cpp

```diff
@@ -15,6 +15,7 @@
 #include "controls/devices/joystick.h"
 #include "controls/devices/kbcontroller.h"
 #include "controls/game_controls.h"
+#include "engine/demomode.h"

```

Comment:
Probably not needed any more?

Reply:
Yes that's true. Thanks for finding it. 🙂 Fixed